gpu4pyscf/lib/cupy_helper/CMakeLists.txt

```diff
@@ -33,38 +33,4 @@ set_target_properties(cupy_helper PROPERTIES
   LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR})
 find_package(CUDAToolkit)
 
-# based on https://github.com/NVIDIA/CUDALibrarySamples/blob/master/cuTENSOR/CMakeLists.txt
```

Comment:
is cutensor not needed?

Reply:
cutensor is still needed as a library. But the line 'find_package(CUTENSOR)' in CMakeLists.txt is unnecessary.

Reply:
Your PR removes these code

Reply:
Right, they are unnecessary either based on my test.